instrumentation/spring-rabbit/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<groupId>io.zipkin.brave</groupId>
+		<artifactId>brave-instrumentation-parent</artifactId>
+		<version>4.16.3-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>brave-instrumentation-spring-rabbit</artifactId>
+	<name>Brave Instrumentation: Spring AMQP (RabbitMQ)</name>
+
+	<properties>
+		<main.basedir>${project.basedir}/../..</main.basedir>
+		<main.java.version>1.6</main.java.version>
+		<main.signature.artifact>java16</main.signature.artifact>
+		<spring-amqp.version>1.7.5.RELEASE</spring-amqp.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.amqp</groupId>
+			<artifactId>spring-amqp</artifactId>
+			<version>${spring-amqp.version}</version>
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.amqp</groupId>
+			<artifactId>spring-rabbit</artifactId>
+			<version>${spring-amqp.version}</version>
+			<scope>compile</scope>
+		</dependency>
+
+		<!-- spring-aop is required for the consumer side instrumentation point -->
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-aop</artifactId>
+			<version>${spring4.version}</version>
+			<scope>compile</scope>
```

Comment:
this probably should be provided scope else you'll override the version a user chooses. same thing for spring-rabbit's dep